bot.py

```diff
@@ -10,12 +10,14 @@
 from geopy.geocoders import GoogleV3
 from math import radians, sqrt, sin, cos, atan2
 
+
 class PokemonGoBot(object):
 
     def __init__(self, config):
         self.config = config
         self.pokemon_list=json.load(open('pokemon.json'))
         self.item_list=json.load(open('items.json'))
+        self.log = logging.getLogger(__name__)
```

Comment:
There's no need for this line here. `self.log` is assigned later on the `_setup_logging` function.